docker/scripts/install-docker-compose-ubuntu.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -e 
+
+# check user privileges:
+if [ "$(whoami)" != 'root' ]; then
+    echo "Super-user privileges are required. Please execute it with 'sudo'."
+    exit 1
+fi
+
+# Install kernel extra's to enable docker aufs support
+# sudo apt-get -y install linux-image-extra-$(uname -r)
+
+# Add Docker PPA and install latest version
+# sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 36A1D7869245C8950F966E92D8576A8BA88D21E9
+# sudo sh -c "echo deb https://get.docker.io/ubuntu docker main > /etc/apt/sources.list.d/docker.list"
+# sudo apt-get update
+# sudo apt-get install lxc-docker -y
+
+# Alternatively you can use the official docker install script
+# wget -qO- https://get.docker.com/ | sh
```

Comment:
Can we remove these? We shouldn't leave dead or commented code (unless there's a good reason and explained close to that code) ;-)